static/scss/home.scss

```diff
@@ -0,0 +1,155 @@
+@import "variables.scss";
+@import "learnmore2.scss";
+@import "foundation";
+
+.featured_books {
+    margin-top: 18px;
+    font-size: $font-size-header;
+}
+
+.module-title {
+    font-size: $font-size-header;
+}
+
+.js-main-container {
+        margin-left: auto;
+        margin-right: auto;
+        max-width: 960px;
+}
+
+#user-block1 {
+	#block-intro-text {
+		float:left;
+		width:702px;
+		font-size: $font-size-header;
+	}
+
+	a#readon {
+		font-size: $font-size-larger;
+	}
+}
+
+
+@include breakpoint(small only) {
+    #js-maincol-fl {
+        display: flex;
+        flex-direction: column;
+    }
+    #js-topsection {
+        visibility: hidden;
+        height: 0px;
+    }
+
+    .book-list > div{
+        justify-content: center;
+    }
+
+    .more_featured_books {
+        display:none;
+    }
+
+    .more_featured_books_mobile {
+        width:100%;
+        > a {
+            width: 100%;
+        }
+    }
+}
+
+@include breakpoint(medium up) {
+    #js-maincol-fl {
+        display: flex;
+        #js-main-continer {
+            flex: 1 84%;
+        }
+
+        #js-rightcol {
+            flex: 1 16%;
+        }
+    }
+
+    .book-list > div {
+    }
+
+    .more_featured_books {
+        padding-left:14px;
+        > i {
+            margin-top:150px;
+            font-size: 26px;
+        }
+    }
+
+    .more_featured_books_mobile{
+        display: none !important;
+    }
+}
+
+#js-main-container {
+    display: flex;
+    flex-direction: column;
+
+    .book-detail {
+        .book-cover {
+            a {
+            }
+        }
+    }
+    .spacer {
+    }
+
+    .featured_books {
+
+    }
+    div {
+    }
+
+}
+
+#js-rightcol {
+    .js-rightcol-padd {
+        background-color: $pale-blue;
+        padding: 10px;
+    }
+}
+
+.jsmodule {
+    margin-bottom: 14px;
+    .google_signup {
+        a {
+            width: 100%;
+            border-radius: 0px;
+            border: none;
+        }
+    }
+}
+
+.sidebar_button {
+    margin-bottom: 10px;
+    input {
+        background-color: $green;
+        color: white;
+        width: 100%;
+        padding: 12px;
+        border: none;
+    }
+
+}
+
+.book-description {
+    overflow: auto;
+}
+
+.book-detail {
+    display: flex;
+
+}
+
+.book-list > div{
+    display: flex;
+    flex-wrap: wrap;
+    box-sizing: content-box;
+}
+
+.thewholebook {
+   border: none !important;
```

Comment:
Can we avoid using `!important`? Maybe find out what other class is adding that border and remove it?